app/controllers/posts_controller.rb

```diff
@@ -4,6 +4,28 @@ def index
     @user = User.find(params[:user_id])
   end
 
+  def new
+    post = Post.create
+    respond_to do |format|
+      format.html { render :new, locals: { post: } }
+    end
+  end
+
+  def create
+    user = current_user
+    post = Post.new(params.require(:post).permit(:title, :text))
```

Comment:
- **[OPTIONAL]** Please add the params method as private for a clean code.  [Lesson link about the strong parameters](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides).